pkg/apis/pipeline/v1alpha1/pipelinerun_types.go

```diff
@@ -89,14 +92,37 @@ type PipelineRef struct {
 	APIVersion string `json:"apiVersion,omitempty"`
 }
 
-// PipelineParamsRef can be used to refer to a specific instance of a Pipeline.
-// Copied from CrossVersionObjectReference: https://github.com/kubernetes/kubernetes/blob/169df7434155cbbc22f1532cba8e0a9588e29ad8/pkg/apis/autoscaling/types.go#L64
-type PipelineParamsRef struct {
-	// Name of the referent; More info: http://kubernetes.io/docs/user-guide/identifiers#names
-	Name string `json:"name"`
-	// API version of the referent
+// AllResultTargetTypes is a list of all ResultTargetTypes, used for validation
+var AllResultTargetTypes = []ResultTargetType{ResultTargetTypeGCS}
+
+// ResultTargetType represents the type of endpoint that this result target is,
+// so that the controller will know how to write results to it.
+type ResultTargetType string
+
+const (
+	// ResultTargetTypeGCS indicates that the URL endpoint is a GCS bucket.
+	ResultTargetTypeGCS = "gcs"
+)
+
+// Results tells a pipeline where to persist the results of runnign the pipeline.
+type Results struct {
+	// Runs is used to store the yaml/json of TaskRuns and PipelineRuns.
+	Runs ResultTarget `json:"runs"`
+
+	// Logs will store all logs output from running a task.
+	Logs ResultTarget `json:"logs"`
```

Comment:
while we're at it, why don't we simplify this and get rid of `Tests` and `Runs`? Discussions with @tejal29 indicated that we might treat Tests like a type of resource instead, and it's not clear how we'll use `Runs`

(sorry for the scope creep!!)